src/sentry/models/identity.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    ArrayField, BoundedPositiveIntegerField, EncryptedJsonField,
+    FlexibleForeignKey, Model
+)
+
+
+# TODO(dcramer): pull in enum library
```

Comment:
[we have enum](https://github.com/getsentry/sentry/blob/master/requirements-base.txt#L15)

Reply:
@EvanPurkhiser yeah i think its more 'it doesnt work out of the box' but i forget.. I have these comments spread around everywhere in these diff uses